tests/jest/disclosure-manager/disclosureCallbacks.test.js

```diff
@@ -0,0 +1,52 @@
+import {
+  addCallback,
+  removeCallback,
+  closeMostRecentDisclosure,
+  getActiveDisclosureCount,
+} from '../../../src/disclosure-manager/_disclosureCallbacks';
+
+const callback1 = () => 1;
+const callback2 = () => 2;
+const callback3 = () => 3;
+
+describe('disclosureCallbacks', () => {
```

Comment:
Not thrilled with these tests, but had to workaround not having the callback array directly exposed.